docs/models.rst

```diff
@@ -246,6 +246,47 @@ this value is perfectly known from the dust map. Therefore, when using
 a function such as `~sncosmo.fit_lc` to fit the parameters, be sure *not* to
 include ``'mwebv'`` in the list of parameters to vary.
 
+Adding color dependant scatter model
+====================================
+
+The intrinsic scattering of SNe Ia is color dependant it can be modelled for simulation purpose
+by G10 or C11 models. The implemention is based on Kessler et al. 2012.
+They both act as random variation in the spectra model of a `~sncosmo.SALT2Source` or `~sncosmo.SALT3Source`.
+
+The G10 model take a `SALTSource` as an argument and can be added to your `~sncosmo.Model` as:
+
+.. code:: python
+
+    >>> source = 'salt2'
+    >>> SALTSource = sncosmo.models.get_source(name=source)
+    >>> G10 = snc.models.G10(SALTsource=SALTSource)
```

Comment:
This document currently assumes `import sncosmo` rather than `import sncosmo as snc`.